src/components/Garden/DecisionDetail/DecisionDetail.js

```diff
@@ -5,6 +5,7 @@ import {
   BackButton,
```

Comment:
What do you think of reusing the conviction proposals logic and including the forum comments related to the link if any?

Reply:
ah yeah good idea